app/src/main/AndroidManifest.xml

```diff
@@ -1,32 +1,34 @@
-<manifest
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="ar.com.wolox.android">
 
-    <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
     <application
         android:name=".example.BootstrapApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:theme="@style/AppTheme">
-
         <activity
             android:name=".example.ui.example.ExampleActivity"
+            android:screenOrientation="portrait"/>
+
+        <activity
+            android:name=".example.ui.viewpager.ViewPagerActivity"
+            android:screenOrientation="portrait" />
```

Comment:
If we are not using ExampleActivity and ViewPagerActivity please erase them in the project. If you need them as an example you can go back through commits